src/components/Button/Button.scss

```diff
@@ -191,6 +192,61 @@ $spinner-size: rem(20px);
   }
 }
 
+.monochrome {
```

Comment:
If we did go with the API where `Button monochrome` renders an outline button, does it let us use less CSS??